src/collaborative_platform/notebooks/Untitled.ipynb

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
That's not very descriptive title.

Reply:
I see. Will change it in the future as making notebooks work with docker is a hassle
and I run short on time right now.

Reply:
I will need to check what it is about and whether is still relevant for the project.